data/json/recipes/recipe_deconstruction.json

```diff
@@ -5092,6 +5092,15 @@
     "components": [ [ [ "felt_patch", 100 ] ], [ [ "thread", 8 ] ] ],
     "flags": [ "BLIND_EASY" ]
   },
+  {
+    "result": "bundle_nylon",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "skill_used": "fabrication",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "nylon", 50 ] ], [ [ "thread", 8 ] ] ],
+    "flags": [ "BLIND_EASY" ]
+  },
```

Comment:
Don't do that. Make the recipe reversible. If you give it a dedicated uncraft like this for some reason people will be able to transform sinew/plant fiber/yarn into thread.

Reply:
Oh dang. All the other bundles have dedicated uncrafts, so I just went with consistency.

Making it reversible is much better. Fixing now.

Reply:
Yeah old bundles started ages ago and then everyone just copied the same method. But reversible is the better solution - you can even make it take less than tying it together (I've been doing it for reversible recipes for some bags)